app/templates/components/nav-bar.hbs

```diff
@@ -45,8 +45,8 @@
         <a href="{{href-to 'login'}}" class="item">{{t 'Login'}}</a>
       {{/if}}
       {{#if (not-eq routing.currentRouteName 'create')}}
-        <div class="item">
-          {{#link-to 'create' class='ui blue button'}}
+        <div class="item ui blue">
```

Comment:
Instead of a link-to inside a div, just have a link-to with the class `item ui blue`

Reply:
![image](https://user-images.githubusercontent.com/24489162/29749259-1632cbe6-8b45-11e7-9415-cefd49143648.png)
sir making whole class link-to is not making 'create event' blue in color due to css property.

Reply:
Sir the changes i earlier made using div tag performs and looks sames as this link: https://pr-786-open-event-frontend-fossasia.surge.sh

Reply:
`ui blue` class does not exist. In this, it will look blue even without the class. Because blue is the default color of an anchor tag in semantic-ui.

But the issue here is, you're not able to click on any other place other than the text. Because you have put an anchor tag with a `div.item`. 

So, please don't have the `link-to` inside a div instead move the link-to outside of div add a class `ui blue` and style it blue via css.